samples/tab-sso/nodejs/README.md

```diff
@@ -233,6 +233,4 @@ For more information see the [Code of Conduct FAQ](https://opensource.microsoft.
 contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
 ## Further Reading.
-[Extend Teams apps across Microsoft 365](https://learn.microsoft.com/en-us/microsoftteams/platform/m365-apps/overview)
```

Comment:
And in the steps of "run the app locally". the steps require user to start ngrok, and npm install, this is not the usage of Teams Toolkit. I kind of recommand to rewrite the README file using Teams Toolkit and use our template file: https://github.com/OfficeDev/TeamsFx-Samples/blob/zhany-readme-template/README_template.md